statsmodels/tsa/vector_ar/irf.py

```diff
@@ -88,9 +88,9 @@ def cum_effect_cov(self, *args, **kwargs):
         raise NotImplementedError
 
     def plot(self, orth=False, impulse=None, response=None,
-             signif=0.05, plot_params=None, subplot_params=None,
-             plot_stderr=True, stderr_type='asym', repl=1000,
-             seed=None, component=None):
+             signif=0.05, plot_params=None, figsize=(10, 10),
```

Comment:
Need a `,*,` to change order.  I would place the `,*,` after orth so that all of the parameters are keyword only except orth.

Reply:
Thanks, I have included it now.